src/config.c

```diff
@@ -3129,6 +3129,8 @@ standardConfig static_configs[] = {
     /* Size_t configs */
     createSizeTConfig("hash-max-listpack-entries", "hash-max-ziplist-entries", MODIFIABLE_CONFIG, 0, LONG_MAX, server.hash_max_listpack_entries, 512, INTEGER_CONFIG, NULL, NULL),
     createSizeTConfig("set-max-intset-entries", NULL, MODIFIABLE_CONFIG, 0, LONG_MAX, server.set_max_intset_entries, 512, INTEGER_CONFIG, NULL, NULL),
+    createSizeTConfig("set-max-listpack-entries", NULL, MODIFIABLE_CONFIG, 0, LONG_MAX, server.set_max_listpack_entries, 128, INTEGER_CONFIG, NULL, NULL),
+    createSizeTConfig("set-max-listpack-value", NULL, MODIFIABLE_CONFIG, 0, LONG_MAX, server.set_max_listpack_value, 64, INTEGER_CONFIG, NULL, NULL),
```

Comment:
we discussed this in a core-team meeting.
we decided that since we don't know how the existing defaults were tuned, and we don't know what to set for the new defaults, we'll take a conservative approach of using 128, so that the performance impact compared to the previous releases is lower (and also the memory benefits).

we decided that even if we some day want to add more "generic" controls on all of these, it should be in a separate PR were we handle all these tunables, so we leave it out for now.